cmd/jaeger/ingester-remote-backend-storage.yaml

```diff
@@ -0,0 +1,50 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [kafka, otlp, jaeger, zipkin]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+  telemetry:
+    metrics:
+      address: 0.0.0.0:8889
+    logs:
+      level: debug
+
+extensions:
+  jaeger_query:
+    trace_storage: some_storage
+
+  jaeger_storage:
+    backends:
+      some_storage:
+        memory:
```

Comment:
this isn't remote storage, but it might still be ok for the integration test purposes. Just adjust the fine name accordingly.